hibernate-core/src/main/java/org/hibernate/cfg/AvailableSettings.java

```diff
@@ -343,6 +343,14 @@
 	String DATASOURCE ="hibernate.connection.datasource";
 
 	/**
+	 * Instructs Hibernate to skip the autocommit check for local transactions since
+	 * the underlying {@link javax.sql.DataSource} has already disabled autocommit.
+	 *
+	 * @since 5.2.10
+	 */
+	String SKIP_AUTOCOMMIT_CHECK ="hibernate.connection.skip_autocommit_check";
```

Comment:
See my comments on docs above.  Again, unless I am mistaken, this applies to non-DataSource providers as well.  Also, I prefer to JavaDoc-link related settings here in `AvailableSettings` .